src/Symfony/Component/Validator/Constraints/GroupSequenceProvider.php

```diff
@@ -23,4 +23,10 @@
 #[\Attribute(\Attribute::TARGET_CLASS)]
 class GroupSequenceProvider
 {
+    public ?string $provider = null;
+
+    public function __construct(array $options = [], string $provider = null)
```

Comment:
```suggestion
    public function __construct(array|string $options = [], string $provider = null)
```

then if $option is a string, make it the provider? Would allow #[GroupSequenceProvider('foo')]

Reply:
As there was no constructor before, there is no BC need to support passing an array.

this could use `public function __construct(string $provider = null)` as constructor, using the `@NamedArgumentConstructor` annotation of doctrine/annotations to support annotations and `HasNamedArguments` attribute of the validator component to support the config file loaders.

Reply:
Updated! Thanks to both of you.